public/migrate.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<script>
+  ;(function () {
+    const allowedOrigins = [
+      'http://localhost:3000',
+      'https://migrate.web-core.pages.dev',
+      'https://web-core.pages.dev',
+      'https://beta.safe.global',
+      'https://app.safe.global',
+    ]
+    // Send the entire localStorage if the origin is allowed
+    window.addEventListener('message', function (event) {
+      if (allowedOrigins.includes(event.origin)) {
+        event.source.postMessage(
+          {
+            type: 'migration-data',
+            payload: JSON.parse(JSON.stringify(localStorage)),
```

Comment:
Probably we should only export / send the data prefixed with `SAFE__` as that's the relevant data we need.

Reply:
Also the keys starting with `immortal`. I think it's better to send everything, and decide what to pick on the receiving end.